hashtagsv2/hashtags/static/js/graph.js

```diff
@@ -118,4 +123,16 @@ $.ajax({
     complete: function(){
         $('#loader3').hide();
     }
-})
\ No newline at end of file
+})
+
+setTimeout(function() {
+    let projects_link = document.getElementById("projectsChart").toDataURL("image/jpeg");
+    let users_link = document.getElementById("usersChart").toDataURL("image/jpeg")
+    let edits_link = document.getElementById("timeChart").toDataURL("image/jpeg")
+    document.getElementById("projects_url").href = projects_link;
+    document.getElementById("projects_url").setAttribute('download', 'top_projects.jpg')
+    document.getElementById("users_url").href = users_link;
+    document.getElementById("users_url").setAttribute('download', 'top_users.jpg')
+    document.getElementById("time_url").href = edits_link;
+    document.getElementById("time_url").setAttribute('download', 'edits_over_time.jpg')
+}, 100);
```

Comment:
Each chart may take different time to load and also the charts are asynchronous. So, setting a time explicitly to call the function is not a good idea and may result in an inappropriate download. Instead you can include `animation` option in each chart such that each function is called after the animation completes.

For example, for the Users chart, you can add an option:

`animation: {
                        onComplete: function(){
                            let users_link = document.getElementById("usersChart").toDataURL("image/jpeg");
                            document.getElementById("users_url").href = users_link;
                            document.getElementById("users_url").setAttribute('download', 'top_users.jpg');
                        }
         }`

and simliarly you can do for all the three charts.